site/src/components/form.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Form
+description: The form related components in the Atlas Design System
+template: standard
+---
+
+# Form
+
+Forms contain form control elements such as the following:
+
+- [`.button`](./button.md)
+- [`.checkbox`](./checkbox.md)
+- [`.help`](./help.md)
+- [`.input`](./input.md)
+- [`.label`](./label.md)
+- [`.radio`](./radio.md)
+- [`.select`](./select.md)
+- [`.textarea`](./textarea.md)
+- custom form elements
+
+## Form Field
+
+The `.field` component is a container for form control elements to provide consistency.
+It can be used to hold `.field-section` containers, which holds the individual elements.
+
+`.field-section-label` is used on the label. Adding `field-section-label-sm`/`field-section-label-lg` makes the label a little smaller or bigger than the default.
+
+```html
+<div class="field">
+	<div class="field-section">
+		<label class="field-section-label field-section-label-sm label" for="sample-form-input-sm">
+			Input
+		</label>
+		<input
+			id="sample-form-input-sm"
+			name="title"
+			class="input"
+			type="text"
+			placeholder="e.g., Why is the sky blue?"
+		/>
+	</div>
+</div>
+<div class="field">
+	<div class="field-section">
+		<label class="field-section-label label" for="sample-form-input"> Input </label>
+		<input
+			id="sample-form-input"
+			name="title"
+			class="input"
+			type="text"
+			placeholder="e.g., Why is the sky blue?"
+		/>
+	</div>
+</div>
+<div class="field">
+	<div class="field-section">
+		<label class="field-section-label field-section-label-lg label" for="sample-form-input-lg">
+			Input
+		</label>
+		<input
+			id="sample-form-input-lg"
+			name="title"
+			class="input"
+			type="text"
+			placeholder="e.g., Why is the sky blue?"
+		/>
+	</div>
+</div>
+<div class="field display-flex">
+	<div class="field-section">
+		<button class="button button-primary margin-right-xxs">Submit</button>
+		<button class="button button-danger">Cancel</button>
+	</div>
```

Comment:
Thinking the form buttons should use flex/utilities or a dedicated class rather than merging with field.

```html
<div class="display-flex gap-xxs">
  <button class="button button-primary">Submit</button>
  <button class="button button-danger">Cancel</button>
</div>
```

Reply:
Updated, but moved further down to the form validation section.